### DISCUSSION TITLE
calibrate xdrip freestyle libre 3 web follower

### DISCUSSION THREAD

**Discussion by mateoSP2014** [NightscoutFoundation/xDrip#2824] [General]
Good afternoon.

Does anyone know if it is possible to calibrate the xdrip in web follower mode with freestyle 3?

thanks

**Comment by PhilTU** [comment]
I believe not, the idea is you're displaying a value from another system and the timing etc is not known to xDrip+

**Reply by mateoSP2014** [reply]
thanks PhilTU, i have tested and doesn´t work.

i have tried with juggluco and i got it.

thanks for your answer
